README.md

```diff
@@ -1386,6 +1386,7 @@ Software for [monitoring](https://en.wikipedia.org/wiki/Monitoring#Computing) sy
 
 _Related: [Wikis](#wikis)_
 
+- [Benotes](https://github.com/fr0tt/benotes) - An open source self hosted notes and bookmarks taking web app. `MIT` `PHP`
```

Comment:
The homepage link should be https://benotes.org/, and https://github.com/fr0tt/benotes should be used as Source Code link

Reply:
@mzch ping

Reply:
Oh, sorry for my delay. I'll fix it soon.

Reply:
Fixed.